### ISSUE TITLE
Specifying done sectors: CLI --> protocol/storage/miner.go code

### ISSUE THREAD

**Issue by ZenGround0** [filecoin-project/go-filecoin#2998]
### Description
As we land #2948 and #2889 the miner actor on the state machine is ready to take in a done set as a parameter to submitPost.  However we will not yet have a way for miner operator users to specify which sectorIDs they want to be finished with.  We should have a simple CLI endpoint `miner sectors rm`, or something like that, which collects user input and passes the information to the storage miner protocol code.

Within the storage miner protocol there should be an endpoint that adds sectorIDs to a datastructure holding all of the sectorIDs that should be deleted.  This collection should then be read during the `submitPoSt` call to generate the user specified `done` input to the submitPoSt message.

### Acceptance criteria
- One CLI command that batches sectorID removals
- Code to handle batching removals in the storage protocol
* Bonus, a `miner sectors refresh` (or something with a better name) that flushes the removed sector ids out of the set in case an operator changes their mind.  Probably this should be its own story motivated by use cases before we start it, but it feels like a friendly addition.

### Where to begin
See the [submitPoSt](https://github.com/filecoin-project/go-filecoin/blob/master/protocol/storage/miner.go#L762) method.

Storing all the sectorIDs to remove directly in an [IntSet](https://github.com/filecoin-project/go-filecoin/blob/master/types/intset.go#L8) seems like the way to go, especially considering the submitPoSt message's `done` param is just a serialized IntSet.  The IntSet interface should be friendly for this story too.

The obvious concurrency model of having only one reader or writer to the "to-be-removed" sectorID collection should be necessary and sufficient.

**Comment by anorth** [comment]
I'm putting this in the backlog for now, but @acruikshank it's probably worth pulling in for the 0.4 release.
